website/src/pages/index.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useEffect, useState } from 'react';
+import { CSSTransition } from 'react-transition-group';
+import clsx from 'clsx';
+import Layout from '@theme/Layout';
+import Link from '@docusaurus/Link';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import styles from './index.module.css';
+import SilkFeatures from '../components/mainPage/SilkFeatures/SilkFeatures';
+import BackgroundLighBalls from '../components/mainPage/LightBalls/LightBalls';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCode } from '@fortawesome/free-solid-svg-icons/faCode';
+import { faGithubAlt } from '@fortawesome/free-brands-svg-icons/faGithubAlt';
+import ReadyForAnything from '../components/mainPage/ReadyForAnything/ReadyForAnything';
+import AllInOneSolution from '../components/mainPage/AllInOneSolution/AllinOneSolution';
+import RealiableAndDependable from '../components/mainPage/ReliableDepend/ReliableDepend';
+import JoinTheConversation from '../components/mainPage/JoinConversation/joinConversation';
+import TryStride from '../components/mainPage/TryStride/TryStride';
+
+
+export default function Home(): JSX.Element {
+  const {siteConfig} = useDocusaurusContext();
+  return (
+
+    <Layout
+      title={`${siteConfig.title}`}
+      description="Description will go into a meta tag in <head />">
+
+      <HomepageHeader />
+
+      <MicrosoftLogo />
```

Comment:
Please keep this directly below the buttons so it is visible to the user. Also maybe not call it a Microsoft logo as that's not what it is.